python/init_container.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status, or zero if all
+# commands in the pipeline exit successfully.
+set -e
+
+# ###
+# 1. section: Define functions
+# ###
+function initialize() {
+  echo "Initializing diagnostics..."
+
+  rm -rf /root/.dcos && mkdir /root/.dcos
+  cd /dcos-cli-directory || exit
```

Comment:
What the point of `|| exit` here with `set -e`?

Reply:
you are right, removed